projects/netgrif-components/schematics/_commons/view-class-info.ts

```diff
@@ -67,6 +67,14 @@ export class ViewClassInfo extends ImportToAdd {
                 return 'Dashboard';
             case 'treeCaseView':
                 return 'TreeCaseView';
+            case 'publicTaskView':
+                return 'PublicTaskView';
+            case 'publicWorkflowView':
+                return 'PublicTaskView';
+            case 'publicSingleTaskView':
+                return 'publicSingleTaskView';
```

Comment:
The first two public views are written with capital 'P', the second two entries are written with regular 'p'. Is this intended?

Reply:
fixed